src/atomate2/vasp/sets/core.py

```diff
@@ -800,6 +808,129 @@ def get_kpoints_updates(
         return {"reciprocal_density": self.reciprocal_density}
 
 
+@dataclass
+class MDSetGenerator(VaspInputGenerator):
+    """
+    Class to generate VASP molecular dynamics input sets.
+
+    Parameters
+    ----------
+    ensemble
+        Molecular dynamics ensemble to run. Options include `nvt`, `nve`, `npt`,
+        and `nph`.
+    start_temp
+        Starting temperature. The VASP `TEBEG` parameter.
+    end_temp
+        Final temperature. The VASP `TEEND` parameter.
+    nsteps
+        Number of time steps for simulations. The VASP `NSW` parameter.
+    time_step
+        The time step (in femtosecond) for the simulation. The VASP `POTIM` parameter.
+    spin_polarized
+        Whether to do spin polarized calculations. The VASP `ISPIN` parameter.
+    """
+
+    ensemble: str = "nvt"
+    start_temp: float = 300
+    end_temp: float = 300
+    nsteps: int = 1000
+    time_step: int = 2
+    spin_polarized: bool = False
+
+    def get_incar_updates(
+        self,
+        structure: Structure,
+        prev_incar: dict = None,
+        bandgap: float = 0,
+        vasprun: Vasprun = None,
+        outcar: Outcar = None,
+    ) -> dict:
+        """
+        Get updates to the INCAR for a molecular dynamics job.
+
+        Parameters
+        ----------
+        structure
+            A structure.
+        prev_incar
+            An incar from a previous calculation.
+        bandgap
+            The band gap.
+        vasprun
+            A vasprun from a previous calculation.
+        outcar
+            An outcar from a previous calculation.
+
+        Returns
+        -------
+        dict
+            A dictionary of updates to apply.
+        """
+        updates = self._get_ensemble_defaults(self.ensemble)
+
+        # Based on pymatgen.io.vasp.sets.MPMDSet. Changes include:
+        # LREAL: True -> AUTO
+        updates.update(
+            {
+                "ENCUT": None,  # None to use VASP default
```

Comment:
Yes, it is taken from pmg [MPMDSet](https://github.com/materialsproject/pymatgen/blob/1f1056cab8b9358f4d1ac2012f2a5732cd29bcf1/pymatgen/io/vasp/sets.py#L2657). 

I agree with you that this may not be good. The [MVLNPTMDSet](https://github.com/materialsproject/pymatgen/blob/1f1056cab8b9358f4d1ac2012f2a5732cd29bcf1/pymatgen/io/vasp/sets.py#L2676) uses a slightly larger value, 1.5*ENCUT, but still may not be ideal since the ENCUT changes from material to material. I prefer to set it to a const value, but smaller than that in the BaseVaspSet.yaml, like 520.

Reply:
Yeah, I think 520 is good.